modin/pandas/series.py

```diff
@@ -1646,8 +1652,12 @@ def quantile(self, q=0.5, interpolation="linear"):  # noqa: PR01, RT01, D200
         """
         Return value at the given quantile.
         """
-        return super(Series, self).quantile(
-            q=q, numeric_only=False, interpolation=interpolation
+        return self._quantile(
```

Comment:
```suggestion
        return super(Series, self).quantile(
```

Reply:
This isn't working like that, it's failing into infinite recursion because wrong super class is selected in this case, and I cannot change the [MRO](https://www.python.org/download/releases/2.3/mro/) because it would break in other places by picking methods from `BasePandasDataset` instead of special-crafted `SeriesCompat`.

Let me add a comment here explaining the reason.